packages/hotdog/hotdog.spec

```diff
@@ -0,0 +1,68 @@
+%global goproject github.com/bottlerocket
+%global gorepo hotdog
+%global goimport %{goproject}/%{gorepo}
+
+%global gosysrev 4abf325e0275e4ef0bdd441dcf497570f1419ab9
+%global gosysrevshort %(c=%{gosysrev}; echo ${c:0:7})
+
+%global runtimespec 1.0.2
+
+Name: %{_cross_os}hotdog
+Version: 1.0.0
+Release: 1%{?dist}
+Summary: Tool with OCI hooks to run the Log4j Hot Patch in containers
+License: Apache-2.0 and MIT
+URL: https://github.com/awslabs/oci-add-hooks
+Source0: http://local.fix.this/hotdog-main.tar.gz
+Source1: https://github.com/opencontainers/runtime-spec/archive/v%{runtimespec}/runtime-spec-%{runtimespec}.tar.gz
+Source2: https://github.com/golang/sys/archive/%{gosysrev}/sys-%{gosysrevshort}.tar.gz
+
+BuildRequires: %{_cross_os}glibc-devel
+
+%description
+%{summary}.
+
+%prep
+%autosetup -Sgit -n hotdog-main -p1
+%cross_go_setup hotdog-main %{goproject} %{goimport}
+
+mkdir -p GOPATH/src/github.com/opencontainers/runtime-spec
+tar -C GOPATH/src/github.com/opencontainers/runtime-spec -xzf %{SOURCE1} --strip 1
+cp GOPATH/src/github.com/opencontainers/runtime-spec/LICENSE LICENSE.runtime-spec
+
+mkdir -p GOPATH/src/golang.org/x/sys
+tar -C GOPATH/src/golang.org/x/sys -xzf %{SOURCE2} --strip 1
+ls -la GOPATH/src/golang.org/x/sys
+cp GOPATH/src/golang.org/x/sys/LICENSE LICENSE.golang-sys
+
+%build
+# Set CGO_ENABLED=0 to statically link hotdog-hotpath, since it runs inside containers that
+# may not have the glibc version used to compile it
+# Set `GO111MODULE=off` to force golang to look for the dependencies in the GOPATH
+export GOPATH=$PWD/GOPATH
+CGO_ENABLED=0 GO111MODULE=off go build -buildmode=pie -ldflags "-linkmode=external" -o hotdog-hotpatch ./cmd/hotdog-hotpatch
```

Comment:
I don't think we want to force the external linker here if we're not also enabling cgo in order to link with our libc. I'd just use the same command from the hotdog Makefile.